cli/tools/upgrade.rs

```diff
@@ -125,11 +125,19 @@ pub fn check_for_upgrades(cache_dir: PathBuf) {
 
 pub async fn upgrade(upgrade_flags: UpgradeFlags) -> Result<(), AnyError> {
   let old_exe_path = std::env::current_exe()?;
-  let permissions = fs::metadata(&old_exe_path)?.permissions();
+  #[allow(unused_variables)]
```

Comment:
Doesn't seem like we need this?

```suggestion
```

Reply:
I think its needed for Windows, since its not used there

Reply:
`permissions` is used on Windows and `permissions` is created via `metadata` so I think we can remove this.